packages/@react-spectrum/provider/test/Provider.test.js

```diff
@@ -283,4 +283,47 @@ describe('Provider', () => {
       expect(onRender).toHaveBeenCalledTimes(2);
     });
   });
+
+  describe('hooks', () => {
+    const TestComponent = ({testid}) => {
+      const breakpoints = useProviderBreakpoints();
+      const colorScheme = useColorScheme();
+
+      return (<div data-testid={testid} data-color={colorScheme} data-breakpoint={breakpoints['S']} />);
+    };
+
+    test('provide access to color scheme and breakpoints', async () => {
+      const inner = {
+        testId: 'innerTest',
+        colorScheme: 'dark'
+      };
+      const outer = {
+        testId: 'outerTest',
+        colorScheme: 'dark'
+      };
+
+      matchMedia.useMediaQuery(mediaQueryMinSmall);
+      const {getByTestId} = render(<Provider theme={theme} colorScheme={outer.colorScheme}>
+        <TestComponent testid={outer.testId} />
+        <Provider
+          theme={theme}
+          colorScheme={inner.colorScheme}
+          breakpoints={{S: 480, M: 640}}>
+          <TestComponent testid={inner.testId} />
+        </Provider>
+      </Provider>);
+
+      const innerElement = getByTestId(inner.testId);
+      const outerElement = getByTestId(outer.testId);
+
+      const colorFiled = 'data-color';
+      const breakpointFiled = 'data-breakpoint';
+
+      expect(innerElement.getAttribute(colorFiled)).toEqual(inner.colorScheme);
```

Comment:
```suggestion
      expect(innerElement.getAttribute(colorField)).toEqual(inner.colorScheme);
```
small typo on this line and next, and line 319